src/fides/api/ops/models/privacy_experience.py

```diff
@@ -161,6 +161,8 @@ def update(self, db: Session, *, data: dict[str, Any]) -> PrivacyExperience:
 
             PrivacyExperienceHistory.create(db, data=history_data, check_name=False)
 
+        # Cache privacy notices for display
+        self.privacy_notices = self.get_related_privacy_notices(db)
```

Comment:
how does this work?

Reply:
I'm just adding these privacy notices as an attribute to the experience object in memory, just long enough to return it in the API response.  Notices aren't related to privacy experiences directly in the database, so this is just an on-the-fly calculation done.